cli/cmd/devtool/start.go

```diff
@@ -643,7 +644,8 @@ func (s local) runRuntime(ctx context.Context, verbose, reset bool) error {
 	logInfo.Printf("Starting runtime\n")
 	defer func() { logInfo.Printf("Stopped runtime\n") }()
 
-	args := []string{"run", "cli/main.go", "start", stateDirLocal, "--no-ui", "--debug", "--allowed-origins", "http://localhost:3001"}
+	stateDir := lookupDotenv("RILL_DEVTOOL_STATE_DIRECTORY")
+	args := []string{"run", "cli/main.go", "start", stateDir, "--no-ui", "--debug", "--allowed-origins", "http://localhost:3001"}
```

Comment:
Regression: `rill devtool start local` should use the `dev-project` directory.

Reply:
looks like rill always uses `dev-project`, will revert the change

Reply:
done!